pylint/checkers/utils.py

```diff
@@ -238,6 +238,13 @@
     {"_sitebuiltins.Quitter", "sys.exit", "posix._exit", "nt._exit"}
 )
 
+DICT_TYPES = (
```

Comment:
Let's keep this in the variable checkers, we don't want to have checkers be coupled because of this constant and the module name 'utils' is horrible.

Reply:
fair point! the repo uses "utils" a ton. So just have both "DICT_TYPES" in each module regardless of duplication?